store/api.go

```diff
@@ -42,6 +43,32 @@ func (m *MetricInfo) Bits() int {
 	return m.bits
 }
 
+// The group ID of this metric
+func (m *MetricInfo) GroupId() int {
+	return m.groupId
+}
+
+// MetricInfoBuilder creates a brand new MetricInfo instance from scratch.
+// Used for testing.
```

Comment:
I have to construct a metricinfo in a test in the pstore package a
different package

On Wednesday, April 20, 2016, rgooch <notifications@github.com> wrote:

> In store/api.go
> <https://github.com/Symantec/scotty/pull/49#discussion_r60531688>:
>
> > @@ -42,6 +43,32 @@ func (m *MetricInfo) Bits() int {
> >  	return m.bits
> >  }
> >
> > +// The group ID of this metric
> > +func (m *MetricInfo) GroupId() int {
> > +	return m.groupId
> > +}
> > +
> > +// MetricInfoBuilder creates a brand new MetricInfo instance from scratch.
> > +// Used for testing.
>
> If this is just used for testing, why not put it into a *_test.go file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Symantec/scotty/pull/49/files/f85a2bb2f5bfbbab8e372251f20477c316e9a6a3#r60531688>
>